src/main/kotlin/com/strumenta/kolasu/model/Processing.kt

```diff
@@ -229,7 +229,7 @@ fun Node.transformChildrenInPlace(operation: (Node) -> Node) {
                         }
                     }
                 } else {
-                    TODO()
+                    throw UnsupportedOperationException("Only modifications in a List and MutableList are supported, not " + value::class.toString())
```

Comment:
```suggestion
                    throw UnsupportedOperationException("Only modifications in a List and MutableList are supported, not " + value::class.toString())
```

```suggestion
                    throw UnsupportedOperationException("Only modifications in a List and MutableList are supported, not ${value::class.toString()}")
```

Kotlin has string interpolation :)

Reply:
Stupid Intellij didn't suggest it, so I was lazy :-)

Reply:
Aaaand I forgot to do this :-D

Reply:
I changed it and merged it